uc/uc.py

```diff
@@ -20,6 +20,7 @@ def marshalled(self, context):
         company_report = body[0]
         company_report.set('ns1:product', self.product_code)
         company_report.set('ns1:version', '2.1')
+        print(context.envelope.getChild('Body'))
```

Comment:
Logged to stdout? I don't see a problem with printing it directly, since it gives the same result as to create a log handler to stdout and log it... what do you think?